article/app/views/fragments/liveFilter.scala.html

```diff
@@ -0,0 +1,63 @@
+@import _root_.liveblog.LiveBlogPageModel
```

Comment:
this file is called liveFilter but it's more of a navigation control rather than a filter